.github/workflows/package-collect.yml

```diff
@@ -17,103 +17,113 @@ on:
         type: string
 
 jobs:
-  package-rpm:
-    runs-on: [self-hosted, collect]
+  package:
 
     strategy:
+      fail-fast: false
       matrix:
         include:
-          - image: alma8
+          - image: centreon-collect-alma8
             distrib: el8
-          - image: alma9
+            package_extension: rpm
+            runner: collect
+            arch: amd64
+          - image: centreon-collect-alma9
             distrib: el9
-    name: package ${{ matrix.distrib }}
+            package_extension: rpm
+            runner: collect
+            arch: amd64
+          - image: centreon-collect-debian-bullseye
+            distrib: debian-bullseye
+            package_extension: deb
+            runner: collect
+            arch: amd64
+          - image: centreon-collect-debian-bullseye-arm64
+            distrib: debian-bullseye
+            package_extension: deb
+            runner: collect-arm64
+            arch: arm64
+
+    runs-on: ${{ fromJson(format('["self-hosted", "{0}"]', matrix.runner)) }}
+
+    container:
+      image: ${{ vars.DOCKER_INTERNAL_REGISTRY_URL }}/${{ matrix.image }}:${{ inputs.img_version }}
+      credentials:
+        username: ${{ secrets.DOCKER_REGISTRY_ID }}
+        password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
+
+    name: package ${{ matrix.distrib }} ${{ matrix.arch }}
 
     steps:
       - name: Checkout sources
         uses: actions/checkout@v3
 
-      - name: Login to Registry
-        uses: docker/login-action@v2
-        with:
-          registry: ${{ vars.DOCKER_INTERNAL_REGISTRY_URL }}
-          username: ${{ secrets.DOCKER_REGISTRY_ID }}
-          password: ${{ secrets.DOCKER_REGISTRY_PASSWD }}
+      - name: Compile sources
+        run: |
+          sudo pip3 install conan==1.57.0 --prefix=/usr --upgrade
+
+          sudo conan install . -s compiler.cppstd=14 -s compiler.libcxx=libstdc++11 --build=missing
+
+          CMAKE="cmake3"
+          if [ "${{ matrix.package_extension }}" = "deb" ]; then
+            CMAKE="cmake"
+          fi
 
-      - name: package rpm ${{ matrix.image }}
+          sudo $CMAKE \
+                  -G "Ninja" \
+                  -DWITH_TESTING=OFF \
+                  -DWITH_BENCH=ON \
+                  -DWITH_MODULE_SIMU=OFF \
+                  -DCMAKE_INSTALL_PREFIX=/usr \
+                  -DWITH_STARTUP_SCRIPT=systemd \
+                  -DWITH_ENGINE_LOGROTATE_SCRIPT=ON \
+                  -DWITH_USER_BROKER=centreon-broker \
+                  -DWITH_GROUP_BROKER=centreon-broker \
+                  -DWITH_USER_ENGINE=centreon-engine \
+                  -DWITH_GROUP_ENGINE=centreon-engine \
+                  -DWITH_VAR_DIR=/var/log/centreon-engine \
+                  -DWITH_DAEMONS=ON \
+                  -DWITH_CREATE_FILES=OFF \
+                  -DWITH_CONFIG_FILES=ON \
+                  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+                  .
+
+          sudo ninja -j 8
```

Comment:
By default ninja runs the number of CPUs tasks in parallel. Wouldn't it be better than forcing 8 tasks?